flexible-classroom/develop/integrate-flexible-classroom.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: "Integrate Flexible Classroom"
+sidebar_position:  2
+type: docs
+description:
+   How to add Flexible Classroom into your Android app.
+---
+
+import IngegrateFlexibleClassroom from '@docs/shared/flexible-classroom/integrate-flexible-classroom/index.mdx';
+
+export const toc = [{}];
+
+
+This page introduces how to configure the whiteboard and recording features in Flexible Classroom in Agora Console.
```

Comment:
This seems to be the description for Configure Flexible Classroom. Integrate Flexible Classroom's description is: "This page introduces how to add Flexible Classroom to your {platform} app."

Reply:
Nice spot. Updated.